src/config/index.ts

```diff
@@ -117,6 +123,12 @@ const configMap = {
     rawTables: rbdRawTables,
     defaultBoundariesFile: `${DEFAULT_BOUNDARIES_FOLDER}/wca_CHIPC_mar2021_projected_jun2021_simple.json`,
   },
+  sierraleone: {
```

Comment:
Let's use the simpler way to import the config

Reply:
See the translation PR

Reply:
done. thanks